pepper-apis/src/main/resources/db-changes/schema/DDP-5442-auth0-log-event-code.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+
+    <changeSet author="evgeniipr" id="20210113-create-auth0-log-event-code-table">
+        <createTable tableName="auth0_log_event_code">
```

Comment:
We usually use generated primary key columns and avoid using "natural" keys as primary keys.  Can you add a generated `bigint` id column here, use it for the fk from the event table, and change `code` from a primary key to a unique key?

Reply:
Fixed